lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
 <label class="u-field-title" aria-hidden="true">
     <%- title %>
 </label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<span class="sr" for="u-field-value-<%- id %>"><%- screenReaderTitle %></span>
```

Comment:
Note that this is also used on the accounts page, so I don't think an h2 would be good for that reason. On the accounts page, it is supposed to be a peer of the account fields, not a title.